lib/trento/databases/events/database_instance_registered.ex

```diff
@@ -20,6 +20,14 @@ defmodule Trento.Databases.Events.DatabaseInstanceRegistered do
     field :start_priority, :string
     field :system_replication, :string
     field :system_replication_status, :string
+    field :system_replication_site, :string
+    field :system_replication_mode, :string
+    field :system_replication_operation_mode, :string
+    field :system_replication_source_site, :string
+    field :system_replication_tier, :integer
     field :health, Ecto.Enum, values: Health.values()
   end
+
+  def upcast(params, _, 3),
+    do: Map.put(params, "system_replication_tier", 0)
```

Comment:
Very curious about where upcasting to version 2 is 😅

Reply:
In `Trento.Databases.Event.Upcaster.UpcastDatabaseId`